py_neuromodulation/nm_features.py

```diff
@@ -116,3 +122,36 @@ def estimate_features(self, data: "np.ndarray") -> dict:
 
     def get_feature(self, fname: FeatureName) -> NMFeature:
         return self.features[fname]
+
+
+def AddCustomFeature(feature_name: str, new_feature: Type[NMFeature]):
```

Comment:
Amazing @toni-neurosc! Just a tiny comment here, aren't functions usually starting with lower case? I think we keep this style throughout the package